backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -11,8 +11,38 @@ const {
 const {
   app: { notmEndpointUrl: dpsUrl },
 } = require('../../config')
+const { isBlank } = require('../../utils')
 
-module.exports = ({ prisonApi, logError }) => async (req, res) => {
+const fetchStaffName = (context, staffId, prisonApi) =>
+  prisonApi.getStaffDetails(context, staffId).then(staff => formatName(staff.firstName, staff.lastName))
+
+const fetchReasonDescription = (context, assignmentReasonCode, caseNotesApi) =>
+  caseNotesApi
+    .getCaseNoteTypes(context)
+    .then(caseNoteTypes => caseNoteTypes.find(type => type.code === 'MOVED_CELL'))
+    .then(cellMoveTypes => {
+      return cellMoveTypes?.subCodes.map(subType => ({
```

Comment:
Could be inlined:
 .then(cellMoveTypes =>  cellMoveTypes?.subCodes.map(subType => ({
        value: subType.code,
        text: subType.description,
      }))
  )`

`